guide/src/reference/targets.md

```diff
@@ -0,0 +1,69 @@
+# Supported Targets
```

Comment:
I think JS developers are going to be thinking "node or Web or workers or ..." here. It might be worth clarifying somehow. Maybe "Supported Target Triples"? That's not great... do you have any other idea for disambiguating?

Reply:
Ah excellent point, how about "Rust Targets" along with a note at the top saying we're not talking about web workers and such? (although that'll also make good documentation eventually)

Reply:
Sounds great!